ffs/api/api_wallet.go

```diff
@@ -60,6 +60,30 @@ func (i *API) NewAddr(ctx context.Context, name string, options ...NewAddressOpt
 	return addr, nil
 }
 
+// SignMessage signs a message using a managed address.
+func (i *API) SignMessage(ctx context.Context, addr string, message []byte) ([]byte, error) {
+	if !i.isManagedAddress(addr) {
+		return nil, fmt.Errorf("%v is not managed by ffs instance", addr)
+	}
+	res, err := i.wm.Sign(ctx, addr, message)
+	if err != nil {
+		return nil, fmt.Errorf("signing message: %s", err)
+	}
+	return res, nil
+}
+
+// VerifyMessage verifies a message signature of a message from a managed address.
+func (i *API) VerifyMessage(ctx context.Context, addr string, message, signature []byte) (bool, error) {
+	if !i.isManagedAddress(addr) {
```

Comment:
In the description for the PR that `VerifyMessage`:
> verifies signed messages from a wallet address. This wallet address can be any wallet address in the Filecoin network

Seems like you shouldn't enforce the provided address is managed. Would make sense this should be any address.

Reply:
Oh yeah... muscle memory. Will change.